pyincore/globals.py

```diff
@@ -44,3 +44,298 @@
 SCIP_PATH = shutil.which("scip")
 
 DAMAGE_PRECISION = 10
+
+DEFAULT_ALLOWED_DEMANDS = {
```

Comment:
globals doesn't feel like the right place for this. Would this be better in either a new "hazardutil" under "utils" or maybe even somewhere within the models/hazard? It could be part of the "hazard.py" maybe? Thoughts?

Reply:
Yes agree. 
I tried to put it under Hazard class (hazard.py) but failed due to circular dependency. In that case, Hazard import HazardService, and HazardService import Hazard...
So I end up creating another class and put it under hazardservice.py, let me know if that works?